CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## [TBD] - TBD
 ### Changed
 * Minimum supported Java version is now 8 (was 7). See [#108](https://github.com/HotelsDotCom/waggle-dance/issues/108).
+* Increased coverage for waggle-dance-core. See [#25](https://github.com/HotelsDotCom/waggle-dance/issues/25) and [#100](https://github.com/HotelsDotCom/waggle-dance/issues/100).
```

Comment:
We normally only put endiuser facing items in the CHANGELOG, most of them won't really care about changes made to unit tests so I'd remove this.